cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -170,6 +170,47 @@ def test_verify_revert_snapshot(test_vm, provider, soft_assert, register_event,
 
 
 @pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider))
+def test_operations_suspended_vm(test_vm, provider, soft_assert):
+    """Tests snapshot operations on suspended vm
+
+    Metadata:
+        test_flag: snapshot, provision
+    """
+    snapshot1 = new_snapshot(test_vm)
+    snapshot1.create()
+    wait_for(lambda: snapshot1.active, num_sec=300, delay=20, fail_func=sel.refresh)
```

Comment:
**REQUIRED**
I recommend to avoid using `pytest_selenium` in the new code. Please use widgetastic.